Cargo.toml

```diff
@@ -194,3 +194,11 @@ os_pipe = ["io-lifetimes/os_pipe"]
 fs-err = ["io-lifetimes/fs-err"]
 #all-impls = ["async-std", "tokio", "os_pipe", "socket2", "mio", "fs-err"]
 all-impls = ["os_pipe", "fs-err"]
+
+# OS compatability features
+
+# Build for linux 4.11 or later
+linux_4_11 = []
+
+# Build without backcompat
```

Comment:
Similarly, could this say "Enable all optimizations for the latest Linux versions" or so?